guide/docs/faq/extensions.mdx

```diff
@@ -0,0 +1,88 @@
+# Extensions
+
+Questions regarding `disnake.ext.commands` and `disnake.ext.tasks` belong here.
+
+:::note
+
+This is placeholder content that has been taken directly from the [documentation](https://docs.disnake.dev/en/latest/), and inherited from `discord.py`. This section will most likely be rewritten in the future.
```

Comment:
Perhaps don't mention placeholder here? This is actively good content, but it is duplicated from the docs, so something referencing that fact would be good.


> This content is taken directly from [the documentation]....

Reply:
So, just ommit "placeholder" from the blocks?

Reply:
Well, I also might omit that it will be rewritten in the future, I don't think that's necessary to mention.

Reply:
I'm not currently satisfied with the amount of questions covered, and will definitely reorganize them later - I've mentioned it so that people keep it in mind when sharing the link elsewhere, and it maaaybe stops working. :P

Reply:
"Please note this page is a work in progress."

Reply:
"This content has been taken directly from the [documentation](https://docs.disnake.dev/en/latest/faq.html), and inherited from `discord.py`. This section will most likely be rewritten in the future."

Linked directly to the docs's FAQ as well